scripts/da_blending_fv3.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+from netCDF4 import Dataset
+import raymond
+import sys
```

Comment:
Do we need to load a dedicated Python conda environment for all modules needed by the blending Python codes?

Reply:
No special conda environment should be needed for anything--conda isn't allowed on WCOSS2 anyway. This was tested on WCOSS2 and should run as is on there. I think we just need to use python v3. On WCOSS2, I was using python/3.8.6.

Reply:
So the question is whether this code will **only** run WCOSS2? 

Otherwise, we will need to make sure numpy, netCDF4, and customized blending-related modules are available in the current Python environment (whether it is a conda env or other similar things) in any platform (at least NOAA HPCs).

For the blending-related modules (such as raymond, etc), I believe you have to modify the PYTHONPATH env variable in some way in order to import them correctly.  Do you do this manually at WCOSS2 now?

Reply:
You shouldn't have to modify the PYTHONPATH either. All you should have to do (and I do this in exregional_make_ics.sh) is link the *.so files (e.g., raymond.cpython-38-x86_64-linux-gnu.so, etc.) to the same directory that the python script is being called from and then you can import them by using "import raymond" etc.

Once we figure out how to add the compilation of these codes using CMake, I will also want to change the names of the *.so files to exclude the "cpython-38-x86_64-linux-gnu" part so it is just "raymond.so" etc. and is generic between all the different machines.

numpy and netCDF4 *should* be widely available on the NOAA HPCs.

Reply:
NumPy and netCDF4 are not available by default although they are installed by various parties.